src/viewer.c

```diff
@@ -401,27 +404,55 @@ static void zoom_image(const char* params)
     } else if (viewport_scale_def(&ctx.vp, params)) {
         name = params;
     } else {
+        struct str_slice param_slices[2];
+        size_t param_slice_num;
+
+        param_slice_num =
```

Comment:
Please move this line down so that the declarations come first and then the code.